domains/yagnik.json

```
@@ -3,9 +3,8 @@
     "username": "yaagnik-patel",
     "email": "yagnikpatel24818@gmail.com"
   },
-  "proxied": true,
-  "records": {
-    "CNAME": "portfolio-git-main-yaagnik-patels-projects.vercel.app",
-    "TXT": "vc-domain-verify=yagnik.is-a.dev,a847575435db916293d3"
+  "record": {
+    "CNAME": ["portfolio-git-main-yaagnik-patels-projects.vercel.app"],
```

CNAME is a string not an array of String
```suggestion
    "CNAME": "portfolio-git-main-yaagnik-patels-projects.vercel.app",
```